content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -324,18 +325,18 @@ <h3 id="toc-source">Selecting a media source</h3>
 
 <h3 id="toc-security">Security</h3>
 
-<p><code>getUserMedia()</code> can only be called from an HTTPS URL, localhost or a <code>file://</code> URL. Otherwise, the promise from the call will be rejected. <code>getUserMedia()</code> also won't work for cross origin calls from iframes: see <a href="https://goo.gl/EuHzyv" title="Chromium Project: Deprecating Permissions in Cross-Origin Iframes">Deprecating Permissions in Cross-Origin Iframes</a> for more detail.</p>
+<p><code>getUserMedia()</code> can only be called from an HTTPS URL, localhost or a <code>file://</code> URL. Otherwise, the promise from the call is rejected. <code>getUserMedia()</code> also doesn't work for cross-origin calls from iFrames. For more information, see <a href="https://goo.gl/EuHzyv" title="Chromium Project: Deprecating Permissions in Cross-Origin Iframes">Deprecating Permissions in Cross-Origin Iframes</a>.</p>
```

Comment:
'iframes' not 'iFrames': we use all-lower-case for element names.

Reply:
Ah okay. My mistake.